packages/wrangler/src/api/worker.ts

```diff
@@ -86,6 +86,15 @@ interface CfDurableObject {
   script_name?: string;
 }
 
+/**
+ * A Service.
+ */
+interface CfService {
+  name: string;
+  script_name: string;
```

Comment:
Given this is a service binding, I think `service_name` would be more fitting.

Reply:
I commented on the wrong line, meant for `packages/wrangler/src/config.ts`

Reply:
good call, yes 

but also, we'll probably have to the so the same with durable objects bindings at some point

Reply:
how do you feel about calling it `service`?

(in q1, once we have the toml rewriter, let's take a crack at just updating the entire config to be consistent and better)

Reply:
made the change so you can see what it looks like

Reply:
`service` looks good too